static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,62 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
```

Comment:
shall we only allow https?

Reply:
We need to allow both as we have no control which protocol our GHE customers use. For instance, our own internal instance uses http so it's safe to assume we're not the only GHE account holders with the same setup.